types/smartystreets-javascript-sdk/index.d.ts

```diff
@@ -128,137 +140,580 @@ export namespace core {
+        cities: City[];
+        /** A [list of ZIP Codes](https://smartystreets.com/docs/cloud/us-zipcode-api#zipcodes) that match the input */
+        zipcodes: ZipCode[];
     }
     interface City {
+        /** The name of the city */
         city: string;
+        /** The official, two-letter state abbreviation */
         stateAbbreviation: string;
+        /** The state name */
         state: string;
+        /** A boolean value indicating whether or not the city name is an approved USPS mailing name */
         mailableCity: string;
     }
     interface ZipCode {
+        /** The 5-digit ZIP Code */
         zipcode: string;
+        /**
+         * The type of ZIP Code. Possible values:
+         * - **S** - Standard (regular ZIP Code)
+         * - **M** - Military (APO/FPO military ZIP Code. Also includes DPO - Diplomatic addresses)
+         * - **P** - P.O. Box (serves only post-office boxes)
+         * - **U** - Unique (belongs primarily to a firm
+         */
         zipcodeType: string;
+        /** A string containing the default city name for this ZIP Code */
         defaultCity: string;
+        /** [The county FIPS code](https://smartystreets.com/articles/county-fips-codes) */
         countyFips: any;
+        /**
+         * The name of the [county in which the ZIP Code is located](https://smartystreets.com/articles/county-by-zip-code)
+         *
+         * Note: The county name listed here pertains to the 5-digit ZIP Code, not necessarily the city.
+         */
         countyName: string;
+        /** The approximate latitude geo-coordinate */
         latitude: number;
+        /** The approximate longitude geo-coordinate */
         longitude: number;
+        /**
+         * Indicates the precision of the latitude and longitude values.
+         * - **None** — Coordinates not known. Reasons could include: address is invalid, military address (APO or FPO), lat/lon coordinates not available.
+         * - **Zip5** — Accurate to a 5-digit ZIP Code level (least precise)
+         * - **Zip6** — Accurate to a 6-digit ZIP Code level
+         * - **Zip7** — Accurate to a 7-digit ZIP Code level
+         * - **Zip8** — Accurate to an 8-digit ZIP Code level
+         * - **Zip9** — Accurate to a [9-digit ZIP Code](https://smartystreets.com/articles/zip-4-code) level (most precise but NOT [rooftop level](https://smartystreets.com/docs/geocoding-accuracy))
+         */
         precision: string;
+        /** The official, two-letter state abbreviation */
         stateAbbreviation: string;
+        /** The state name */
         state: string;
-        alternateCounties: [County];
+        /**
+         * The county FIPS codes, county names, state abbreviations, and states, that share the same zip code.
+         *
+         * Note: The county names listed here pertain to the 5-digit ZIP Code, not necessarily the city.
+         * @example
+         * Input: zipcode=42223
+         * Output: [
+         * 	{
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/smartystreets-javascript-sdk/index.d.ts;
  Line: 702;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.